src/m365/spo/commands/web/web-installedlanguage-list.ts

```diff
@@ -0,0 +1,69 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+import { WebInstalledLanguagePropertiesCollection } from './WebPropertiesCollection';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+}
+
+class SpoWebInstallEdlanguageListCommand extends SpoCommand {
+  public get name(): string {
+    return commands.WEB_INSTALLEDLANGUAGE_LIST;
+  }
+
+  public get description(): string {
+    return 'Lists all installed languages on site';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['DisplayName', 'LanguageTag', 'Lcid'];
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    if (this.verbose) {
+      logger.logToStderr(`Retrieving all installed languages on site ${args.options.webUrl}...`);
+    }
+
+    const requestUrl: string = `${args.options.webUrl}/_api/web/RegionalSettings/InstalledLanguages`;
```

Comment:
Since we're using the URL in just one place without further modifications, we can define it directly in `requestOptions` without using a separate const